config/config/src/main/java/io/helidon/config/internal/FilesystemWatchPollingStrategy.java

```diff
@@ -117,7 +117,13 @@ private void fireEvent(WatchEvent<?> watchEvent) {
                 });
     }
 
-    void initWatchServiceModifiers(WatchEvent.Modifier... modifiers) {
+    /**
+     * Add modifiers to be used when registering the {@link WatchService}.
+     * See {@link Path#register(WatchService, WatchEvent.Kind[],
+     * WatchEvent.Modifier...) Path.register}.
+     * @param modifiers the modifiers to add
+     */
+    public void initWatchServiceModifiers(WatchEvent.Modifier... modifiers) {
```

Comment:
Our `module-info.java` for config contains:

`    exports io.helidon.config.internal to io.helidon.config.git;`

so, among other things, this class (in Java 9 and later) is not visible outside the config module itself (except to our git module, obviously). I don't think we want to unconditionally export all (or any) of a package called `internal`.

`PollingStrategies` in `io.helidon.config` is pretty much the outward interface config exposes to the world for change notification. Do we need to expose the ability to set `Modifier`s by adding a new method to  `PollingStrategies.FileSystemWatchBuilder`, for example?

Reply:
I've create issue #202 to track this.